growthbook/growthbook
# Add Group endpoints for SCIM

### Features and Changes


Adds endpoints for SCIM groups. Currently these endpoints are optimized for Okta and may not work well with other idP's. 



### Dependencies

#1657 
#1640 


### Testing



1. Setup an Okta dev account if you don't have one already
2. Setup SSO and set an enterprise key for your local instance of GrowthBook
3. Use a tool like ngrok to expose your local APIs to Okta
4. Setup a SCIM test app in Okta and try pushing groups with users*
5. Try removing users from the groups*
6. Try removing a group from the SCIM test app*

* Any of these steps should trigger a change in the `teams` collection if a group is being removed or added and in the `organizations` members array if someone is being added or removed to a group

### To-Do

- [x] Add `managedByIdp` to `TeamModel`
